Directory.Packages.props

```diff
@@ -16,6 +16,7 @@
     <!-- Test Platform, .NET Test SDK and Object Model  -->
     <MicrosoftNETTestSdkVersion>17.12.0</MicrosoftNETTestSdkVersion>
     <MicrosoftPlaywrightVersion>1.47.0</MicrosoftPlaywrightVersion>
+    <MicrosoftTestingExtensionsFakesVersion>17.12.0-beta.24429.1</MicrosoftTestingExtensionsFakesVersion>
```

Comment:
Is there any stable version of fakes available (now or near future)?

Reply:
Yep we have created a ticket with @drognanar for that.